slsa_for_models/README.md

```diff
@@ -0,0 +1,37 @@
+# SLSA for Models
+
+To protect the supply chain of traditional software against tampering (like in
+the [Solarwinds attack][solarwinds]), we can generate [SLSA][slsa] provenance,
+for example by using the [SLSA L3 GitHub generator][slsa-generator].
+
+This projects shows how we can use the same generator for training models via
+GitHub Actions. While most of the ML models are too expensive to train in such a
+fashion, this is a proof of concept to prove that _the same traditional software
+supply chain protections can be applied to ML_. Future work will involve
+covering training ML models that require access to accelerators (i.e., GPUs,
+TPUs) or that require multiple hours for training.
```

Comment:
Perhaps add a reference to https://www.kubeflow.org/docs/components/pipelines/v1/sdk/pipelines-with-tekton/ and  https://github.com/tektoncd/chains which could be combined to achieve similar outcomes?

Reply:
Thanks for the reminder! Was planning to include it when finalizing the entire SLSA readme, but it's better to actually add here in the intro.